docs/se/guides/40_webclient.adoc

```diff
@@ -0,0 +1,416 @@
+    -Dpackage=io.helidon.examples.quickstart.se
+----
+
+It created a new directory named `helidon-quickstart-se`.
+
+[source,bash]
+.Open this directory
+----
+cd helidon-quickstart-se
+----
+
+The Helidon quickstart is a greeting application supporting several HTTP requests such as GET and PUT. Using it will
+be time saving and allow us to focus on the WebClient features and how to use it.
+
+==== Modify pom dependency
+
+In the pom.xml, remove the test scope under the WebClient dependency as it will be used not only in the tests.
+
+[source,xml]
+.Remove test scope:
+----
+<dependency>
+    <groupId>io.helidon.webclient</groupId>
+    <artifactId>helidon-webclient</artifactId>
+    <scope>test</scope> <!--  Remove this line -->
+</dependency>
+----
+
+==== Add ClientExample class
+
+In `io.helidon.examples.quickstart.se` package, create a new class named ClientExample. This class will use the
+WebClient to send request to the greeting application.
+
+[source,java]
+.Create ClientExample class:
+----
+package io.helidon.examples.quickstart.se;
+
+public class ClientExample {
+
+    public static void main(String[] args) {
+
+    }
+}
+----
+
+Add the following code to create a WebClient instance. The builder approach allow us to create the WebClient with
+specific settings. It improves the readability and simplicity of the code.
+
+[source,java]
+.Add WebClient instance to the main method:
+----
+import io.helidon.media.jsonp.JsonpSupport;
+import io.helidon.webclient.WebClient;
+
+...
+
+WebClient webClient = WebClient.builder()
+                .baseUri("http://localhost:8080")   // <1>
+                .addMediaSupport(JsonpSupport.create())     // <2>
+                .build();
+----
+<1> Uri from which all request will be executed.
+<2> Register a support for Jsonp.
+
+By default, Helidon quickstart run on the host localhost and port 8080. If you changed these values in configuration
+files, make sure to modify them in the Uri as well.
+Once build, the WebClient can be used to send a GET request to the greeting application.
+
+[source,java]
+.Send a GET request to the target endpoint:
+----
+webClient.get()     // <1>
+    .path("/greet")     // <2>
+    .request(String.class)  // <3>
+    .peek(System.out::println)  // <4>
+    .await();   // <5>
+----
+<1> Create a Http GET request.
+<2> Path to reach after the base uri provided during instantiation.
+<3> Execute the request and return `Single` of the desired type.
+<4> Print the response in the console.
+<5> Wait for server response because of reactive approach.
+
+The path method joins `/greet` to the WebClient base Uri. The target Uri for this request becomes
+`http://localhost:8080/greet` where the response should be a greeting message. As the response will be printed in the
+console, a String is more convenient than the default return type for the request. The default type is WebClientResponse
+which contains response code, headers and non-handled entity.
```

Comment:
Received response entity will be automatically handled as a `String`. If no specific type is put into the method `request()`, `WebClientResponse` is returned by default. This `WebClientResponse` object contains response code, headers and non-handled entity.